### ISSUE TITLE
Try to get Leifer data through the pipeline

### ISSUE THREAD

**Issue by Charles-Fieseler-Vienna** [Zimmer-lab/wbfm#954]

**Comment by Charles-Fieseler-Vienna** [comment]
Actually looking at my training code, as long as I have the centroids loaded in intermediate_global_tracks, then I should be able to calculate the training data... not sure about the accuracy though.

Key function: get_bbox_data_for_volume

**Comment by Charles-Fieseler-Vienna** [comment]
Seems good from an unpacked project! Next step: calculating accuracy measures given that I can't really calculate traces

**Comment by Charles-Fieseler-Vienna** [comment]
Currently barlow_tracking needs the segmentation metadata, but the alt_barlow_tracking rule doesn't... not sure if I can really run it (should test)
